portal-impl/src/com/liferay/portlet/asset/service.xml

```diff
@@ -101,7 +101,7 @@
 		<reference entity="AssetEntry" package-path="com.liferay.portlet.asset" />
 		<reference entity="AssetVocabulary" package-path="com.liferay.portlet.asset" />
 	</entity>
-	<entity deprecated="true" local-service="true" name="AssetCategoryProperty" remote-service="true">
+	<entity deprecated="true" local-service="true" name="AssetCategoryProperty" remote-service="false">
```

Comment:
@dantewang @ChiZelin 
This entity has been deprecated, why not remove the entity? Are there any usages left?

Tina.

Reply:
@tinatian Yes, there are. AssetCategoryPropertyLocalServiceUtil is used by AssetUtil.